semeio/jobs/scripts/fm_pyscal.py

```diff
@@ -172,8 +172,7 @@ def run(
     parameters are explicit, they are implicit here, and gathered
     from parameters.txt"""
     if not os.path.exists(relperm_parameters_file):
-        _logger.error("%s does not exist", relperm_parameters_file)
-        sys.exit(1)
+        raise FileNotFoundError(f"{relperm_parameters_file} does not exist")
```

Comment:
Probably a good, change, but can you also catch it in `fm_pyscal` and `sys.exit(str(err))` (or something like that), otherwise my error message from the command line becomes:
```
Traceback (most recent call last):
  File "/.../semeio/bin/fm_pyscal", line 33, in <module>
    sys.exit(load_entry_point('semeio', 'console_scripts', 'fm_pyscal')())
  File "/.../semeio/semeio/jobs/scripts/fm_pyscal.py", line 36, in main_entry_point
    run(
  File "/.../semeio/semeio/jobs/scripts/fm_pyscal.py", line 203, in run
    (int_param_wo, int_param_go) = _get_interpolation_values(
  File "/.../semeio/semeio/jobs/scripts/fm_pyscal.py", line 261, in _get_interpolation_values
    raise ValueError(
ValueError: Do not include brackets in parameter name <INTERPOLATE_WO>
```

Reply:
That might be a correct error message, depending on your command line arguments?

But, I will revert this change, all the other `if`s in this function end in `sys.exit(1)` so better keep it consistent.

Reply:
The message is perfectly fine, I would just like to avoid the traceback.